### DISCUSSION TITLE
Labelling and numbering equations inside \begin{align} \end{align} latex environment.

### DISCUSSION THREAD

**Discussion by raghu-mahajan** [quarto-dev/quarto-cli#9511] [Q&A] [answered]
### Description

I am trying to use Quarto to build a website that has pages with lots of equations. I am a mathematical physicist and a heavy user of latex. I would like to be able to use the \begin{align} and \end{align} latex environment. I am rendering all pages in html. The rendering works fine. However, the equation numbers are not generated.

For example, I have the following block in a .qmd file

````qmd
\begin{align}
H &= - \frac{1}{2} \frac{d^2}{d x^2} \label{hamiltonian} \\
\psi_n(x) &=  \exp\left(i  \frac{n}{R} x \right) \frac{1}{\sqrt{2\pi R}} \,,\quad n \in \mathbb{Z} \label{wavefunctions} \\
E_n &= \frac{1}{2} \,  \frac{n^2}{R^2}\,. \label{energylevels}
\end{align}
````

I didn't include the rest of my .qmd file, but the full output looks like that in the following image. Note the missing equation numbers.

<img width="1424" alt="Screenshot 2024-04-27 at 7 29 44 PM" src="https://github.com/quarto-dev/quarto-cli/assets/17094269/51feff24-d89b-40b3-aa50-23337d000db3">

I am using a basic website format as explained [here](https://ucsb-meds.github.io/creating-quarto-websites/).
My _quarto.yml file looks like this

````qmd
project:
  type: website
  output-dir: docs

website:
  title: "String Theory and Quantum Gravity"
  navbar:
    left:
      - href: index.qmd
        text: Home
      - about.qmd

format:
  html:
    theme: cosmo
    css: styles.css
    toc: true
    page-layout: full

editor: visual
````

How do I get Quarto to automatically generate equation numbers when using the latex align environment? 

I know that using the double-dollar signs to write and label equations works if I supply labels using {#eq-abc}, but I would really like to use the full functionality of latex, since my equations will get quite complicated and run over multiple lines.

Thank you!

**Comment by mcanouil** [comment, marked as answer]
I am redirecting to the issue that is discussing it:
- https://github.com/quarto-dev/quarto-cli/issues/2275
